networklibrary/src/main/java/com/eis/smsnetwork/SMSNetDictionary.java

```diff
@@ -0,0 +1,78 @@
+package com.eis.smsnetwork;
+
+
+import com.eis.communication.network.NetDictionary;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Concrete implementation of a NetDictionary
+ *
+ * @author Marco Cognolato
+ */
+public class SMSNetDictionary implements NetDictionary<String, String> {
+
+    Map<String, String> dict = new HashMap<>();
+
+    /**
+     * Adds a resource to the network dictionary
+     *
+     * @param key      The key which defines the resource
```

Comment:
As said before, maybe it would be better to specify what resource is defined by the key.
Same in the next method.